ckanext/scheming/helpers.py

```diff
@@ -433,3 +433,12 @@ def scheming_flatten_subfield(subfield, data):
         for k in record:
             flat[prefix + k] = record[k]
     return flat
+
+
+@helper
```

Comment:
The goal on https://github.com/ckan/ckan/issues/7016 was that extensions didn't have to add their own `csrf_input()` helpers. After we add the two mock helpers on the 2.8 and 2.9 branches, `{{ h.csrf_input() }} ` won't fail (see https://github.com/ckan/ckan/pull/7021#pullrequestreview-1081899586).

So this should be merged after the next 2.8 and 2.9 patch releases are out

Reply:
> So this should be merged after the next 2.8 and 2.9 patch releases are out

polite reminder